datadog_lambda/wrapper.py

```diff
@@ -155,12 +162,21 @@ def __init__(self, func):
         except Exception as e:
             logger.error(format_err_with_traceback(e))
 
+    def _get_blocking_response(self):
+        if not config.appsec_enabled:
```

Comment:
I think we might not need this method at all. In its 3 usages, 2 actually have the `config.appsec_enabled` right before calling it anyway. The only place left is in the `BlockingException` case, which implies that appsec already enabled? if not just add the check there? 
in short, I think we can just call `get_asm_blocked_response(self.event_source)` directly.